src/pages/basic_data/institution_admin/FindPersonPanel.tsx

```diff
@@ -85,14 +85,18 @@ export const FindPersonPanel = () => {
                   {({ field, meta: { touched, error } }: FieldProps<string>) => (
                     <TextField
                       {...field}
+                      type="number"
```

Comment:
Kanskje droppe denne? Øk/mink er ikke relevant for NIN, og ser ut til at det blir ball ved feilmelding også

```suggestion
```
![bilde](https://github.com/user-attachments/assets/185d067c-33c3-41cb-8873-2f7fa14cbf1f)